ext/drcontainers/hashtable.h

```diff
@@ -153,6 +153,8 @@ hashtable_init(hashtable_t *table, uint num_bits, hash_type_t hashtype, bool str
  * @param[in]  cmp_key_func        A callback for comparing two keys.
  *   Leave it NULL if no callback is needed and the default is to be used.
  *   For HASH_CUSTOM, a callback must be provided.
+ *
```

Comment:
Mention explicitly above that this one uses closed addressing.

Reply:
Done.